### DISCUSSION TITLE
Is there any difference between carbon--type-style and type-style?

### DISCUSSION THREAD

**Discussion by Yohanna** [carbon-design-system/carbon#8524] [Help] [answered]
Just wanted to confirm if there is any difference between `carbon--type-style` and `type-style`?

```scss
@include carbon--type-style('productive-heading-03');

// vs.

@include type-style('body-long-01');
```

Also, is the `@carbon/type` package's [README](https://github.com/carbon-design-system/carbon/blob/main/packages/type/README.md) the best place to find documentation on how to import different styles in Carbon, is there a different page I should be looking at?

Thanks.

**Comment by joshblack** [comment, marked as answer]
Hi there @Yohanna! 👋 

Great question, if you're using `carbon-components` directly then we ship a mixin called `type-style` as an alias for `carbon--type-style`. They should be completely interchangeable.

As we move towards Sass Modules in future versions of Carbon, you can expect all namespaced mixins to drop the `carbon--` and become just the mixin name itself, as well.

Hope this helps!
